Jobs.Fetcher.TikTok/Helpers/ApiDataFetcher.cs

```diff
@@ -0,0 +1,242 @@
+                       Album = musicJSON["album"].ToString()
+            };
+        }
+
+        public static List<Challenge> GetTikTokChallengesFromJson(JToken challengesJSON) {
+            var newChallenges = new List<Challenge>();
+            if (challengesJSON == null) {
+                return newChallenges;
+            }
+            foreach (var challenge in challengesJSON) {
+                newChallenges.Add(new Challenge() {
+                    Id = challenge["id"].ToString(),
+                    Title = challenge["title"].ToString(),
+                    Description = challenge["desc"].ToString(),
+                    ProfileThumbnail = challenge["profileThumb"].ToString(),
+                    ProfileMedium = challenge["profileMedium"].ToString(),
+                    ProfileLarge = challenge["profileLarger"].ToString(),
+                    CoverThumbnail = challenge["coverThumb"].ToString(),
+                    CoverMedium = challenge["coverMedium"].ToString(),
+                    CoverLarge = challenge["coverLarger"].ToString(),
+                    IsCommerce = challenge["isCommerce"].ToObject<bool>()
+                });
+            }
+            return newChallenges;
+        }
+
+        public static List<Tag> GetTikTokTagsFromJSON(JToken tagsJSON) {
+            var newTags = new List<Tag>();
+            if (tagsJSON == null) {
+                return newTags;
+            }
+            foreach (var tag in tagsJSON) {
+                newTags.Add(new Tag() {
+                    AweMeId = tag["awemeId"].ToString(),
+                    Start = tag["start"].ToObject<int>(),
+                    End = tag["end"].ToObject<int>(),
+                    HashtagName = tag["hashtagName"].ToString(),
+                    HashtagId = tag["hashtagId"].ToString(),
+                    Type = tag["type"].ToObject<int>(),
+                    UserId = tag["userId"].ToString(),
+                    IsCommerce = tag["isCommerce"].ToObject<bool>(),
+                    UserUniqueId = tag["userUniqueId"].ToString(),
+                    SecureUId = tag["secUid"].ToString(),
+                    SubType = tag["subType"].ToObject<int>()
+                });
+            }
+            return newTags;
+        }
+
+        public static List<EffectSticker> GetTikTokEffectStickersFromJSON(JToken effectStickersJSON) {
+            var effectStickers = new List<EffectSticker>();
+            if (effectStickersJSON == null) {
+                return effectStickers;
+            }
+            foreach (var sticker in effectStickersJSON) {
+                var newEffectSticker = new EffectSticker() {
+                    Name = sticker["name"].ToString(),
+                    Id = sticker["ID"].ToString()
+                };
+                effectStickers.Add(newEffectSticker);
+            }
+            return effectStickers;
+        }
+
+        public static Author GetTikTokAuthorFromAuthorJSON(JToken authorJSON) {
```

Comment:
When I'm coding, I often mix "JSON" and "Json", like you did here. Do you think we could establish a standard for the project?

Reply:
I think having a standard is always a good idea. I just checked the rest of the project and it seems that 'Json' is more common, maybe because Newtonsoft uses this pattern. So I'll change everything to "Json"